docs/section19/19-7.md

```diff
@@ -0,0 +1,87 @@
+
+**Rule Assertion:** Options are Pass/Fail/UNDETERMINED      
+**Appendix G Section:** G3.1.2.5 and Exceptions 1-4           
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None    
+
+**Evaluation Context:** Each HeatingVentilatingAirConditioningSystem  
+
+**Applicability Checks:**  None
+   
+**Function Calls:**  
+1. get_dict_of_zones_and_terminal_units_served_by_hvac_sys()    
+2. match_data_element()  
+3. get_component_by_id()  
+4. get_list_hvac_systems_associated_with_zone()  
+5. get_min_OA_CFM_sch_zone()  
+6. aggregate_min_OA_schedule_across_zones()  
+
+
+
+## Rule Logic:   
+- Create dictionary of hvac systems and associated zones and terminal units for the baseline: `dict_of_zones_and_terminal_units_served_by_hvac_sys_b = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(B_RMI)`   
+- For each hvac system in the B_RMI: `for hvac in B_RMI...HeatingVentilatingAirConditioningSystem:`  
+    - Reset was_DCV_modeled_baseline boolean variable to false: `was_DCV_modeled_baseline = false`  
+    - Reset was_DCV_modeled_proposed boolean variable to false: `was_DCV_modeled_proposed = false`  
+    - Reset zone_air_distribution_effectiveness_greater_than_1 boolean to false: `zone_air_distribution_effectiveness_greater_than_1 = false`  
+    - Reset hvac_system_serves_only_labs boolean variable to true: `hvac_system_serves_only_labs = true`  
+    - Check if demand controlled ventilation was modeled at the fan_system: `if hvac.fan_system.demand_control_ventilation_control == true: was_DCV_modeled_baseline= true`  
+    
+    - Get list of zones served by the hvac system: `zones_list_hvac_sys_b = dict_of_zones_and_terminal_units_served_by_hvac_sys_b[hvac.id]["ZONE_LIST"]`  
+    - Set counter equal to 0: `counter = 0`  
+    
+    - For each zone associated with the HVAC system: `for zone in zones_list_hvac_sys_b:`  
+        - For each space associated with the zone, check if any spaces are NOT labs: `for space in zone.spaces:`
+            - Check if the lighting space type is NOT a lab: `if space.lighting_space_type not in [LABORATORY_EXCEPT_IN_OR_AS_A_CLASSROOM]: hvac_system_serves_only_labs = false`    
+        
+        - Add each zone's OA CFM schedule to the list of zone OA CFM schedules for the baseline: `zone_OA_CFM_list_of_schedules_b[counter] = get_min_OA_CFM_sch_zone(B_RMI, zone.id)`  
+        - Get the analogous zone object from the proposed RMI (if this is not strictly the correct syntax please advise Weili): `zone_p = get_component_by_id(P_RMI,match_data_element(P_RMI,Zone,zone.id))`    
+        - Add each zone's OA CFM schedule to the list of zone OA CFM schedules for the proposed: `zone_OA_CFM_list_of_schedules_p[counter] = get_min_OA_CFM_sch_zone(P_RMI, zone_p.id)`  
+
+        - For each terminal unit serving the zone, check if DCV was modeled: `for terminal in zone.terminals:`  
```

Comment:
@claperle this might already be be the case but just confirming- in some cases a zone might be served by 2 HVAC systems, example- DOOAS + HP. in this case, the terrminals associated with the DOAS would have DCV but not the heat pump. 
I think this is already addressed by looking through each terminal serving a zone, but line 44- _For each zone associated with the HVAC system: `for zone in zones_list_hvac_sys_b_:`  makes me think we're looping through HVAC systems and not through zones.

Reply:
@supriyagoel I think the way it is setup works because the logic is cycling through the **baseline** HVAC systems, identifying all zones served, and then looking at ALL of the terminals associated with each of these zones in the proposed design (so it would catch all HVACs serving the zones in the proposed). If any terminals or their associated HVAC systems have DCV in the proposed design then the RDS considers DCV to have been modeled in the proposed.  Please let me know if I am missing something.

Reply:
yes, that makes sense. No, I dont think you're missing anything.